application/__init__.py

```diff
@@ -0,0 +1 @@
+VERSION = "0.0.1"
```

Comment:
Should be `__version__` as per https://www.python.org/dev/peps/pep-0396/#specification